packages/react/src/components/Authenticator/withAuthenticator.tsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
 import { Authenticator, AuthenticatorProps } from './Authenticator';
+import { UseAuthenticator } from './hooks/useAuthenticator';
```

Comment:
Since this is only used as a type, could we make that clear by using `import type`?
```suggestion
import type { UseAuthenticator } from './hooks/useAuthenticator';
```